### PR TITLE
ImageList controller creates an ImageJob object

### LINKED ISSUE #11: create ImageList controller

### PR DESCRIPTION
**Pull request by AldairCoronel**
Add ImageList functionality to create an ImageJob object.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,9 @@
+<<<<<<< HEAD
```
Comment (sozercan) [inline .gitignore]: some merge issues

File: .github/workflows/workflow.yaml [hunk 2]
```diff
@@ -34,3 +34,5 @@ jobs:
 
       - name: Unit test
         run: make test
+
```
Comment (sozercan) [inline .github/workflows/workflow.yaml]: extra lines here

File: config/manager/kustomization.yaml [hunk 3]
```diff
@@ -12,5 +12,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: ashnam/controller
```
Comment (sozercan) [inline config/manager/kustomization.yaml:15]: we can change this in #23

File: controllers/imagejob/imagejob_controller.go [hunk 4]
```diff
@@ -38,14 +38,14 @@ func Add(mgr manager.Manager) error {
 
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager) reconcile.Reconciler {
-	return &Reconciler{
```
Comment (sozercan) [inline controllers/imagejob/imagejob_controller.go]: revert Reconciler name changes

Comment (AldairCoronel) [reply controllers/imagejob/imagejob_controller.go]: If I revert the name to ImageJobReconciler I get the following lint error:

`exported: type name will be used as imagejob.ImageJobReconciler by other packages, and that stutters; consider calling this Reconciler (revive)`

**Comment by sozercan** [comment]
continuing on #18

**Pull request closed without merging**